HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -1270,29 +1271,47 @@ def self.process_load_infiltration_ventilation(mj, hpxml_bldg, all_zone_loads, a
     # Calculate ventilation airflow rates
     q_unb_cfm, q_bal_cfm, q_preheat, q_precool, q_recirc, bal_sens_eff, bal_lat_eff = get_ventilation_data(hpxml_bldg)
 
-    # Calculate adjusted infiltration cfm (combined with ventilation)
-    infil_cfm_heat = (icfm_heat**1.5 + q_unb_cfm**1.5)**0.67 - q_unb_cfm
-    infil_cfm_cool = (icfm_cool**1.5 + q_unb_cfm**1.5)**0.67 - q_unb_cfm
+    # Calculate net infiltration cfm (NCFM; infiltration combined with unbalanced ventilation)
+    if q_unb_cfm > 0
+      # Negative pressure, so NCFM = (ICFM^1.5 + CFMimb^1.5)^0.67
+      infil_ncfm_heat = (icfm_heat**1.5 + q_unb_cfm**1.5)**0.67
+      infil_ncfm_cool = (icfm_cool**1.5 + q_unb_cfm**1.5)**0.67
+    else
+      if icfm_heat < q_unb_cfm.abs
+        # Dominating positive pressure, so NCFM = 0
+        infil_ncfm_heat = 0.0
+      else
+        # Mitigating positive pressure, so NCFM = (ICFM^1.5 - ABS(CFMimb)^1.5)^0.67
+        infil_ncfm_heat = (icfm_heat**1.5 - q_unb_cfm.abs**1.5)**0.67
+      end
+      if icfm_cool < q_unb_cfm.abs
+        # Dominating positive pressure, so NCFM = 0
+        infil_ncfm_cool = 0.0
+      else
+        # Mitigating positive pressure, so NCFM = (ICFM^1.5 - ABS(CFMimb)^1.5)^0.67
+        infil_ncfm_cool = (icfm_cool**1.5 - (-q_unb_cfm)**1.5)**0.67
```

Comment:
> It hurts a bit on my brain that the equation for `infil_ncfm_heat` uses `q_unb_cfm.abs` while here it uses `-q_unb_cfm`.

Good catch. I changed the first equation to use abs but forgot to change the second for consistency.

> Basically the equation is just:
> 
> _infil_ncfm_cool = (icfm_cool**1.5 + (q_unb_cfm)**1.5)**0.67,_

I originally wrote the equation like this, but it's not actually correct. Using `(q_unb_cfm)**1.5` with a negative q_unb_cfm will give an error.

But I agree that it can simplified a bit. I will do that. For what it's worth, I'm trying to reasonably match the language in Worksheet E which has this:

![image](https://github.com/NREL/OpenStudio-HPXML/assets/5861765/be81c179-c048-481c-b367-2b5947302193)

Reply:
Oh I didn't know that the negative q_unb_cfm will throw an error on the equation, weird. Yep I checked these equations to make sure our implementation is correct, looks good. 👍